src/main/java/dk/kb/labsapi/SolrExport.java

```diff
@@ -28,11 +28,7 @@
 import org.apache.solr.client.solrj.response.FacetField;
 import org.apache.solr.client.solrj.response.QueryResponse;
 import org.apache.solr.common.SolrDocument;
-import org.apache.solr.common.params.CommonParams;
-import org.apache.solr.common.params.FacetParams;
-import org.apache.solr.common.params.GroupParams;
-import org.apache.solr.common.params.HighlightParams;
-import org.apache.solr.common.params.SolrParams;
```

Comment:
No need to change imports in this project, but could you change settings in your editor to always use explicit imports (`import org.apache.solr.common.params.SolrParams;`) and never use star-imports (`import org.apache.solr.common.params.*;`)